worldedit-core/src/main/java/com/sk89q/worldedit/util/SideEffect.java

```diff
@@ -29,6 +29,7 @@
     ENTITY_AI(State.OFF),
     EVENTS(State.OFF);
 
+    // TODO Make these components in WE8
```

Comment:
Why not just do this now (maybe in separate PR, remove comments here)? It'd be easy to deprecate out.